src/client/Microsoft.Identity.Client/Instance/Discovery/KnownMetadataProvider.cs

```diff
@@ -68,6 +85,12 @@ void AddToKnownCache(InstanceDiscoveryMetadataEntry entry)
             AddToKnownCache(cloudEntryGermanay);
             AddToKnownCache(usGovCloudEntry);
             AddToKnownCache(usCloudEntry);
+            AddToPublicEnvironment(publicCloudEntry);
+        }
+
+        public static bool isKnownPublicEnvironment(string environment)
```

Comment:
PascalCasing and naming (`IsPublicEnvironment` since "known" is already part of the class name)